### PR TITLE
Temp service shutter

### PR DESCRIPTION
**Pull request by EduardoAquinta**

### CODE REVIEW

File: webapp/server.js [hunk 1]
```diff
@@ -117,74 +94,150 @@
...
 }, express.static(path.join(__dirname, './dist/assets/css')));
 
+// app.use(forceHttps);
+
+if (config.SERVICE_UNAVAILABLE === true) {
+  app.all('*', (req, res) => {
+    console.log('Service Unavailable.');
+    res.status('503');
+    res.sendFile(path.join(__dirname, '/app/static/service-unavailable.html'))
+  });
```
[the remark this answers is not in the document]
Comment (monotypical) [reply webapp/server.js:104]: this is fine, we want to read the service unavailable page when the app is in maintenance mode

File: webapp/server.js [hunk 2]
```diff
@@ -117,74 +94,150 @@
...
-  cookie: { httpOnly: true , secure: isSecure},
-  unset: 'destroy'
-}));
+    }),
+    sameSite: 'none',
+    saveUninitialized: false,
+    resave: false,
+    cookie: { httpOnly: true , secure: isSecure},
+    unset: 'destroy'
+  }));
```
[the remark this answers is not in the document]
Comment (monotypical) [reply webapp/server.js:183]: we have not actually changed these request handlers in this PR, we've just changed some whitespace

File: webapp/server.js [hunk 3]
```diff
@@ -117,74 +94,150 @@
...
-});
+  app.get(/^([^.]+)$/, function (req, res, next) {
+    if (config.SERVICE_UNAVAILABLE === true) {
+      console.log('Service Unavailable.');
+      res.status('503');
+      res.sendFile(path.join(__dirname, '/app/static/service-unavailable.html'));
+    } else {
+      matchRoutes(req, res, next);
+    }
+  });
```
[the remark this answers is not in the document]
Comment (monotypical) [reply webapp/server.js:218]: this is fine, we want to read the service unavailable page when the app is in maintenance mode

**Review by monotypical — APPROVED** [review]
LGTM

**Review by bnjn-mt — APPROVED** [review]

**Pull request merged**